.travis.yml

```diff
@@ -58,6 +58,7 @@ env:
     - "TEST_SUITE=spec:legacy DB=postgres"
 
 before_install:
+  - "gem install bundler"
```

Comment:
The bundler for 2.1 on travis is too old (1.7.X) to parse the `optional: true` group definition in the Gemfile. It requires the latest bundler version 1.10. It doesn't seem to add an awful lot of overhead to the travis build though (1-2 seconds).